### PR TITLE
Added: When the app is in the background, hided te contents of the app

### LINKED ISSUE #47: iOS - Hide the contents in recent apps when lockscreen is set
Feature request via email

> After I set Lockscreen, and I swipe up the auth app (iPhone), the contents of the app can be seen like many other apps except password manager (set lockscreen). Like password manager, can the contents be hidden when Lockscreen is set?

### PR DESCRIPTION
**Pull request by i-aiymen**
## Description

Added: Hide the contents of the app, when is it in the background - Both android and ios

In android, it shows a blank white/black background, when in background

In iOS, it shows the launch icon, when in background

photo_2023-12-06_12-53-02

Android Physical Device

photo_2023-12-06_12-52-55

Android stimulator

Also, compared the bitwarden apps in android, the flow is same

iOS Device: Need to verify 
@ua741 @vishnukvmd Could you run on iOS devices and verify it?

Closes #47 

## Type of Change

- [ ] 🖼️ New icon
- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -24,13 +24,28 @@ import "package:flutter/material.dart";
...
+      privacyImageName: "LaunchImage",
+      autoLockAfterSeconds: 5,
+      lockTrigger: IosLockTrigger.didEnterBackground,
+    ),
+    androidOptions: const PrivacyAndroidOptions(
+      enableSecure: true,
+      autoLockAfterSeconds: 5,
+    ),
+    backgroundColor: Colors.white.withOpacity(0),
+    blurEffect: PrivacyBlurEffect.extraLight,
```
Comment (vishnukvmd) [inline lib/main.dart]: Let's set this depending on the app's theme?

File: lib/main.dart [hunk 2]
```diff
@@ -24,13 +24,28 @@ import "package:flutter/material.dart";
...
 final _logger = Logger("main");
 
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
   final savedThemeMode = await AdaptiveTheme.getThemeMode();
   await _runInForeground(savedThemeMode);
+  await PrivacyScreen.instance.enable(
+    iosOptions: const PrivacyIosOptions(
+      enablePrivacy: true,
+      privacyImageName: "LaunchImage",
```
Comment (vishnukvmd) [inline lib/main.dart:38]: Does this asset exist?

File: lib/main.dart [hunk 3]
```diff
@@ -24,13 +24,28 @@ import "package:flutter/material.dart";
...
+      autoLockAfterSeconds: 5,
```
Comment (vishnukvmd) [inline lib/main.dart]: Do we need `autoLock` since we already have a separate lockscreen?

Comment (vishnukvmd) [reply lib/main.dart]: Also, how does this affect the behavior of the app when the existing lock screen is turned on?

File: lib/main.dart (hunk 2 above)
Comment (i-aiymen) [reply lib/main.dart:38]: it's inside the ios folder

Comment (i-aiymen) [reply lib/main.dart:38]: LaunchImage asset was already created in the ios folder

Comment (i-aiymen) [reply lib/main.dart:38]: if its not working, then we have to create it

File: lib/main.dart (hunk 3 above)
Comment (i-aiymen) [reply lib/main.dart]: will remove the autolock. Its irrelevant

**Review by vishnukvmd — APPROVED** [review]

**Pull request merged**